addons/common/functions/fnc_toHex.sqf

```diff
@@ -22,33 +22,18 @@ _number = ((round abs _number) max 0) min 255;
 if (isNil QGVAR(hexArray)) then {
     GVAR(hexArray) = [];
 
-    private _minLength = 2;
-
-    for [{_i = 0;}, {_i < 256}, {_i = _i + 1}] do {
-        private _num = _i;
-        private _hex = ["", "0"] select (_i == 0);
-
-        while {_num > 0} do {
-            private _rest = _num mod 16;
-            _rest = switch _rest do {
-                case 10 : {"A"};
-                case 11 : {"B"};
-                case 12 : {"C"};
-                case 13 : {"D"};
-                case 14 : {"E"};
-                case 15 : {"F"};
-                default {str _rest};
-            };
-            _num = floor (_num / 16);
-            _hex = _rest + _hex;
-        };
-
-        while {count toArray _hex < _minLength} do {
-            _hex = "0" + _hex;
-        };
-
-        GVAR(hexArray) pushBack _hex;
-    };
+    private _digits = [
+        "0","1","2","3","4","5","6","7",
+        "8","9","A","B","C","D","E","F"
+    ];
+
+    {
+        private _x1 = _x;
+
+        {
+            GVAR(hexArray) pushBack (_x1 + _x);
+        } forEach _digits;
+    } forEach _digits;
 };
 
 GVAR(hexArray) select _number // return
```

Comment:
`// return`

Quality comment here

Reply:
I've been using this for forever. Whenever the last statement's return value is the return value of the whole function and it is not something trivial like a single variable, I write // return in addition to leaving out the semi colon. This makes it clear that when someone happens to edit the function for whatever reason, it is not good to add lines below this one. It is kind of obvious here, but not necessarily so. E.g. for functions that do stuff as well as returning something instead of being a pure getter like this one.